test/run_tests.pl

```diff
@@ -82,15 +92,15 @@ BEGIN
     if ($arg eq 'alltests') {
 	warn "'alltests' encountered, ignoring everything before that...\n"
 	    unless $initial_arg;
-	%tests = map { $_ => basename($_) } @alltests;
+	%tests = map { $_ => $_ } @alltests;
     } elsif ($arg =~ m/^(-?)(.*)/) {
```

Comment:
There's something wrong with the indentation here...  could it be that we let TABs sneak in?  Would you mind untabifying?

Reply:
No problem, done.